Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.41873&attempt_id=3bbb3517-6364-4bcc-ba28-65af8581d34b)

Attempted: neovim

<details><summary>Partial log (click to expand)</summary><p>

```
        /build/source/src/nvim/generators/gen_declarations.lua:255: in main chunk
        [C]: at 0x00405660
make[2]: *** [src/nvim/CMakeFiles/nvim.dir/build.make:253: src/nvim/auto/eval/encode.c.generated.h] Error 1
make[1]: *** [CMakeFiles/Makefile2:2865: src/nvim/CMakeFiles/nvim.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[100%] Built target tty-test
make: *** [Makefile:152: all] Error 2
builder for '/nix/store/g76lcg5l56fza9jc4zdb46s088qq45a8-neovim-unwrapped-0.3.0.drv' failed with exit code 2
cannot build derivation '/nix/store/94kj7nh1cd1ksrsqxjr5d7zybiwa4rb3-neovim-0.3.0.drv': 1 dependencies couldn't be built
error: build of '/nix/store/94kj7nh1cd1ksrsqxjr5d7zybiwa4rb3-neovim-0.3.0.drv' failed
```
</p></details>

--- a/pkgs/applications/editors/neovim/default.nix
+++ b/pkgs/applications/editors/neovim/default.nix
@@ -1,4 +1,4 @@
-{ stdenv, fetchFromGitHub, cmake, gettext, libmsgpack, libtermkey
+{ stdenv, fetchFromGitHub, cmake, gettext, libmsgpack, libtermkey, libiconv
 , libtool, libuv, luaPackages, ncurses, perl, pkgconfig
 , unibilium, vimUtils, xsel, gperf, callPackage
 , libvterm-neovim
@@ -32,6 +32,7 @@ let
       luaPackages.lua
       gperf
     ] ++ optional withJemalloc jemalloc
+      ++ optional stdenv.isDarwin libiconv
       ++ lualibs;
 
     nativeBuildInputs = [

